lib/data/repositories/v1/receipt_repository.dart

```diff
@@ -10,20 +12,16 @@ class ReceiptRepository {
 
   /// Retrieves all of the users receipts
   /// This includes both their used tickets and purchased tickets
-  Future<List<Receipt>> getUserReceipts() async {
-    final usedTicketRequest = _api.apiV1TicketsGet(used: true);
-    final purchasesRequest = _api.apiV1PurchasesGet();
-
-    final usedTicketResponse = await usedTicketRequest;
-    final purchaseResponse = await purchasesRequest;
+  Future<Either<ApiError, List<Receipt>>> getUserReceipts() async {
```

Comment:
Any reason this was changed? The original intention was to send both requests out, and then wait for them. Reasoning being, it is the waiting time that will be the longest, so we might as well send all requests first and then wait, since any waiting done at that point should affect both requests. Not just the first one sent.